cumulusci/conftest.py

```diff
@@ -75,3 +78,19 @@ def fallback_orgconfig():
 vcr = fixture(salesforce_vcr, scope="module")
 
 create_task_fixture = fixture(create_task_fixture, scope="function")
+
+
+# TODO: This should also chdir to a temp directory which
+#       can represent the repo-root, but that will require
+#       test case changes.
```

Comment:
Not sure I agree. I think we should try to standardize how we're building project configs for tests and make sure that includes creating a temporary repo root, but I don't think we need one for tests that don't use a project config.

Reply:
If we chdir to a random directory on pytest-startup then we know for sure that we are neither reading or writing from the CumulusCI REPO_ROOT by accident. Any test that wants to opt-in to reading or writing from the CumulusCI REPO_ROOT should do so explicitly.

Reply:
Oh I see you don't mean the repo root of an arbitrary test project, you mean changing the current directory so the repo root of CumulusCI itself doesn't accidentally get treated as a project in tests. Yeah, good idea.